service/src/main/java/bio/terra/workspace/service/petserviceaccount/PetSaService.java

```diff
@@ -179,10 +183,7 @@ public Policy enablePetServiceAccountImpersonation(UUID workspaceId, UserWithPet
       Binding bindingToRemove =
```

Comment:
Is the warning above correct now? Aren't we revoking proxy group access to the pet SA?
(Or am I totally confused?? which is totally possible!)

Same comment/question for the enable call.

Reply:
I think we should remove all the TODOs pointing to PF-991, which is a "Closed, won't do" ticket.

Originally we thought allowing pets to `actAs` themselves would be a temporary fix, so I marked all the places we'd have to remove. That changed and we expect to keep that functionality, so the TODOs are now out of date.